app/models/financialDetails/DocumentDetail.scala

```diff
@@ -96,7 +96,7 @@ case class DocumentDetail(taxYear: String,
     else isPaid
   }
 
-  val isPartPaid: Boolean = outstandingAmount.getOrElse[BigDecimal](0) != originalAmount.getOrElse[BigDecimal](0)
+  val isPartPaid: Boolean = outstandingAmount.getOrElse[BigDecimal](0).abs != originalAmount.getOrElse[BigDecimal](0).abs
```

Comment:
what would happen if one value negative and one positive?
I would rather add unit test to see various scenarios

Reply:
Agree it's a valid question. thanks

Reply:
I've removed the .abs and added a unit tests, thanks